config/rbac/role.yaml

```diff
@@ -55,6 +55,58 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - gateway.networking.k8s.io.kuadrant.io
```

Comment:
is this right?

Reply:
Good catch.
kubebuilder.
I'll fixup in a PR

Reply:
https://github.com/Kuadrant/multi-cluster-traffic-controller/pull/57